docs/guides/security/firewalld-beginners.md

```diff
@@ -38,8 +37,8 @@ You'll need:
 
 * A Rocky Linux machine of any kind, local or remote, physical or virtual
 * Access to the terminal, and a willingness to use it
-* You need root access, or at least the ability to use `sudo` on your user account. For simplicity's sake, I'm assuming all commands are being run as root.
-* A basic understanding of SSH wouldn't hurt for managing remote machines.
+* You need root access, or at least the ability to use `sudo` on your user account. For simplicity's sake, I'm assuming all commands are being run as root
+* A basic understanding of SSH wouldn't hurt for managing remote machines
```

Comment:
Removing the punctuation here is OK. The style guide will let you put a period or an end in in the last item, though, so it isn't technically *wrong* as originally written.